src/main/java/nextstep/ladder/domain/Member.java

```diff
@@ -0,0 +1,22 @@
+package nextstep.ladder.domain;
+
+public class Member {
+    public static final int VALID_LENGTH = 5;
+
+    private String name;
+
+    public String getName(){
+        return name;
+    }
+
+    public void validateNameLength(String name){
+        if (name.length() > VALID_LENGTH){
+            throw new IllegalArgumentException(name + "의 이름이 5글자 이상입니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException(name + "의 이름이 "  + VALID_LENGTH + "글자 이상입니다.");
```
의미를 부여해둔 상수값을 사용해보면 어떨까요? 😄